pkg/platform/platform.go

```diff
@@ -115,6 +116,18 @@ func InfraStatus(client cnoclient.Client) (*bootstrap.InfraStatus, error) {
 			return nil, fmt.Errorf("failed to retrieve HostedControlPlane %s: %v", types.NamespacedName{Namespace: hc.Namespace, Name: hc.Name}, err)
 		}
 		res.HostedControlPlane = hcp
+		if res.HostedControlPlane.Spec.Networking.APIServer == nil {
+			res.HostedControlPlane.Spec.Networking.APIServer = &hyperv1.APIServerNetworking{
+				AdvertiseAddress: pointer.String("172.20.0.1"),
```

Comment:
Why does the backport ignore ipv6?

Reply:
Backport is only for IBM Cloud where we don't have IPV6